bun/bunerrors/errors_internal_test.go

```diff
@@ -0,0 +1,149 @@
+package bunerrors
```

Comment:
Since this is a library let's use `bunerrors_test` package so the tests use the package like a real user would.

Looks like you have some private methods which I'm not sure will work outside the package.